app/code/Magento/InventoryConfiguration/Model/StockConfigurationInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryConfiguration\Model;
+
+interface StockConfigurationInterface
```

Comment:
please add description for this interface

Reply:
better to re-name interface to StockItem configuration, because it's configuration of Product in Stock (in MSI naming that corresponds to Stock Item)

Reply:
Done